packages/@atjson/offset-annotations/src/annotations/cne-audio-embed.ts

```diff
@@ -0,0 +1,21 @@
+import { BlockAnnotation } from "@atjson/document";
+
+export enum AudioEnvironments {
+  Production = "production",
+  Sandbox = "sandbox",
+}
+
+export const CneAudioEnvironments = {
+  [AudioEnvironments.Production]: `https://embed-audio.cnevids.com`,
+  [AudioEnvironments.Sandbox]: `https://embed-audio-sandbox.cnevids.com`,
+};
+
+export class CneAudioEmbed extends BlockAnnotation<{
+  audioEnv: AudioEnvironments;
```

Comment:
You would need to export your enum at the top level of the package if you are going to do this. It's easier if you just make this a union type:
```suggestion
  audioEnv: "production" | "sandbox";
```

Reply:
I am using this as `enum` to don't repeat the reserved words in others areas of the app, ie:
- packages/@atjson/source-html/src/converter/third-party-embeds.ts
- packages/@atjson/renderer-html/src/index.ts

Reply:
Ah I see, you are re-exporting everything here from the annotations index file, and everything there from the top level. Ok, this is all good 👍